enterprise/frontend/src/embedding-sdk/lib/plugins.ts

```diff
@@ -1,18 +1,39 @@
+import type { ReactNode } from "react";
+
+import type { DashCardMenuItem } from "metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenu";
 import type { ClickAction, ClickObject } from "metabase/visualizations/types";
+import type Question from "metabase-lib/v1/Question";
 
 export type SdkDataPointObject = Pick<
   ClickObject,
   "value" | "column" | "data" | "event"
 >;
 
-export type SdkClickActionPluginsConfig = Pick<
-  SdkPluginsConfig,
-  "mapQuestionClickActions"
->;
+export type SdkClickActionPluginsConfig = (
+  clickActions: ClickAction[],
+  clickedDataPoint: SdkDataPointObject,
+) => ClickAction[];
+
+type DashCardMenuCustomElement = ({
```

Comment:
Maybe I went overboard on this one but this basically gives users the option to:

*   use their own menu for dashcards
*   add items to the default dashcard menu
*   remove the default items with the booleans